components/ModEditFacilityForm.vue

```diff
@@ -0,0 +1,290 @@
+            :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityAddressLine2Ja')"
+        />
+    </div>
+
+    <div
+        class="submission-form-section"
+    >
+        <span class="mb-3.5 text-center text-primary-text text-2xl font-bold font-sans leading-normal">
+            {{ $t('modSubmissionForm.googleMapsInformation') }}
+        </span>
+        <ModInputField
+            v-model="submissionFormFields.googlemapsURL.value"
+            data-testid="submission-form-google-maps"
+            :label="$t('modSubmissionForm.labelFacilityGoogleMapsUrl')"
+            type="url"
+            :placeholder="$t('modSubmissionForm.placeholderTextFacilityGoogleMapsUrl')"
+            :required="true"
+            :input-validation-check="validateWebsite"
+            :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityGoogleMapsUrl')"
+            :autofill-value="submissionFormFields.googlemapsURL.value"
+        />
+        <ModInputField
+            v-model="submissionFormFields.mapLatitude.value"
+            data-testid="submission-form-mapLatitude"
+            :label="$t('modSubmissionForm.labelFacilityMapLatitude')"
+            type="text"
+            :placeholder="$t('modSubmissionForm.placeholderTextFacilityMapLatitude')"
+            :required="true"
+            :input-validation-check="validateFloat"
+            :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityMapLatitude')"
+        />
+        <ModInputField
+            v-model="submissionFormFields.mapLongitude.value"
+            data-testid="submission-form-mapLongitude"
+            :label="$t('modSubmissionForm.labelFacilityMapLongitude')"
+            type="text"
+            :placeholder="$t('modSubmissionForm.placeholderTextFacilityMapLongitude')"
+            :required="true"
+            :input-validation-check="validateFloat"
+            :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityMapLongitude')"
+        />
+    </div>
+    <ModHealthcareProfessionalSearchbar data-testid="submission-form-doctor-search" />
+    <button
+        v-show="screenStore.activeScreen === ModerationScreen.EditFacility"
+        type="submit"
+        class="bg-currentColor text-white font-bold py-2 px-4 my-2 rounded w-56"
+    >
+        {{ $t('modSubmissionForm.updateButtonText') }}
+    </button>
+</template>
+
+<script lang="ts" setup>
+import { type Ref, ref } from 'vue'
+import { useModerationScreenStore, ModerationScreen } from '~/stores/moderationScreenStore'
+import { validateAddressLineEn,
+    validateAddressLineJa,
+    validateNameEn,
+    validateNameJa,
+    validatePhoneNumber,
+    validateCityEn,
+    validateEmail,
+    validateFloat,
+    validatePostalCode,
+    validateWebsite,
+    validateCityJa } from '~/utils/formValidations'
+
+const screenStore = useModerationScreenStore()
+
+const submissionFormFields = {
```

Comment:
Let's update these variable names as well.

Reply:
How will it work with `v-model` if it is being used in the Submission or the Facility. How will we be able to choose between the two?

Reply:
@NabbeunNabi I think we should probably move this to a huddle with @ermish. I think I'm also leaning away from trying to have everything as just one form.